diktat-gradle-plugin/src/main/kotlin/org/cqfn/diktat/plugin/gradle/DiktatJavaExecTaskBase.kt

```diff
@@ -45,8 +46,16 @@ open class DiktatJavaExecTaskBase @Inject constructor(
             if (diktatExtension.debug) {
                 add("--debug")
             }
-            add(diktatExtension.inputs.files.joinToString { it.path })
+            diktatExtension.inputs.files.forEach {
+                val pattern = project.trimRootDir(it.path)
+                add("\"$pattern\"")
+            }
+            diktatExtension.excludes?.files?.forEach {
+                val pattern = project.trimRootDir(it.path)
+                add("\"!$pattern\"")
+            }
         }
+        logger.lifecycle("Setting args to $args")
```

Comment:
Actually I don’t like this logging. Say explicitly which arguments you are setting, not just “args”

Reply:
Oh, it was supposed to be debug logging. I'll change the text and set the level to debug.